server/src/test/java/org/elasticsearch/health/HealthServiceTests.java

```diff
@@ -90,7 +87,7 @@ private void assertExpectedHealthIndicatorResults(
             1000,
             getExpectedHealthIndicatorResultsActionListener(onResponseCalled, expectedHealthIndicatorResults)
         );
-        assertBusy(() -> assertThat(onResponseCalled.get(), equalTo(true)));
+        assertBusy(() -> assertTrue(onResponseCalled.get()));
```

Comment:
Do you mind not changing the existing assertions? This way it will be easier to scope the review to the actual changes.

You can still use what you want in your tests, because these need to be reviewed anyway :)